catalyst_voices/lib/widgets/cards/role_chooser_card.dart

```diff
@@ -0,0 +1,241 @@
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:flutter/material.dart';
+
+/// A constant grayscale [ColorFilter] used to de-saturate an image.
+const _grayscaleFilter = ColorFilter.matrix([
+  0.2126,
+  0.7152,
+  0.0722,
+  0,
+  0,
+  0.2126,
+  0.7152,
+  0.0722,
+  0,
+  0,
+  0.2126,
+  0.7152,
+  0.0722,
+  0,
+  0,
+  0,
+  0,
+  0,
+  1,
+  0,
+]);
+
+/// A role chooser card, responsible as a building block
+/// for any role selection, and available in both
+/// interactive and view-only mode.
+class RoleChooserCard extends StatelessWidget {
+  /// The current displaying value.
+  final bool value;
+
+  /// Needs to be a rasterized image.
+  final String imageUrl;
+
+  /// The text label displaying on the card.
+  final String label;
+
+  /// Locks the value and shows it as default, only the selected value appears.
+  final bool lockValueAsDefault;
+
+  /// Hides the "Learn More" link.
+  final bool isLearnMoreHidden;
+
+  /// Toggles view-only mode.
+  final bool isViewOnly;
+
+  /// A callback triggered when the role selection changes.
+  final ValueChanged<bool>? onChanged;
+
+  /// A callback triggered when the "Learn More" link is clicked.
+  final VoidCallback? onLearnMore;
+
+  const RoleChooserCard({
+    super.key,
+    required this.value,
+    required this.imageUrl,
+    required this.label,
+    this.lockValueAsDefault = false,
+    this.isLearnMoreHidden = false,
+    this.isViewOnly = false,
+    this.onChanged,
+    this.onLearnMore,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: 100,
+      decoration: isViewOnly
+          ? null
+          : BoxDecoration(
+              border: Border.all(
+                color: Theme.of(context).colors.outlineBorderVariant!,
+                width: 1,
+              ),
+              borderRadius: BorderRadius.circular(8),
+            ),
+      child: Padding(
+        padding: EdgeInsets.all(isViewOnly ? 8 : 12),
```

Comment:
you can set `padding: EdgeInsets.all(isViewOnly ? 8 : 12)` in `Container`. No need for this widget here